Makefile

```diff
@@ -203,14 +213,19 @@ cmd/query/app/ui/actual/index.html.gz: jaeger-ui/packages/jaeger-ui/build/index.
 	# do not delete dot-files
 	rm -rf cmd/query/app/ui/actual/*
 	cp -r jaeger-ui/packages/jaeger-ui/build/* cmd/query/app/ui/actual/
-	find cmd/query/app/ui/actual -type f | grep -v .gitignore | xargs gzip
+	find cmd/query/app/ui/actual -type f | grep -v .gitignore | xargs gzip --no-name
+	find cmd/query/app/ui/actual -type f | grep -v .gitignore | xargs sha256sum | tee ui.sha256sum.txt
+
 
 jaeger-ui/packages/jaeger-ui/build/index.html:
 	$(MAKE) rebuild-ui
 
 .PHONY: rebuild-ui
 rebuild-ui:
 	cd jaeger-ui && yarn install --frozen-lockfile && cd packages/jaeger-ui && yarn build
+	# Asset manifest must be in sorted order for binary reproducibility.
+	mv jaeger-ui/packages/jaeger-ui/build/asset-manifest.json .asset-manifest.json
+	cat .asset-manifest.json | python -c 'import json, sys; o = json.loads(sys.stdin.read()); print(json.dumps(o, indent=2, sort_keys=True))' > jaeger-ui/packages/jaeger-ui/build/asset-manifest.json
```

Comment:
My preference would be to do this in the UI project using Node.js, so that UI build is fully responsible for producing stable/repeatable artifacts.

Reply:
I made a pass at this and... well I'm kind of at a loss for why it doesn't work.
https://github.com/TripleDogDare/jaeger-ui/pull/1/files

Honestly could probably just delete the file as I don't think it's being used at all.

Reply:
Here's a pass at resolving this in the UI build.
https://github.com/jaegertracing/jaeger-ui/pull/981